src/components/transaction/notification/Notification.tsx

```diff
@@ -23,6 +24,25 @@ export const Notification: React.FC<{
     {},
   )
 
+  //fix for NEP-17 contract that emit transfer events with ByteString as amount
+  for (const notification of notifications) {
+    if (notification.state.type === 'Array') {
+      const isTransfer = notification.event_name === 'Transfer'
+
+      if (isTransfer) {
+        const amountStackItem = notification.state.value[2]
+        if (amountStackItem.type === 'ByteString') {
```

Comment:
Do we need optional chaining here to prevent a type error in case `amountStackItem` is undefined???

`    if (amountStackItem?.type === 'ByteString') {`

Reply:
it cannot be undefined